test-api-generator/citrus-test-api-generator-core/src/test/resources/org/citrusframework/openapi/generator/ExpectedCodeGenIT/expectedgen/rest/extpetstore/spring/ExtPetStoreBeanConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package org.citrusframework.openapi.generator.rest.extpetstore.spring;
+
+import java.util.List;
+
+import org.citrusframework.endpoint.Endpoint;
+import org.citrusframework.openapi.OpenApiRepository;
+import org.citrusframework.openapi.generator.rest.extpetstore.request.ExtPetApi;
+import org.citrusframework.openapi.testapi.ApiActionBuilderCustomizer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import static org.citrusframework.openapi.generator.rest.extpetstore.ExtPetStoreOpenApi.extPetStoreSpecification;
+
+@Configuration
+@jakarta.annotation.Generated(value = "org.citrusframework.openapi.generator.CitrusJavaCodegen", date = "2025-01-29T23:14:48.524898+01:00[Europe/Zurich]", comments = "Generator version: 7.9.0")
+public class ExtPetStoreBeanConfiguration {
+
+    @Bean
+    public OpenApiRepository extPetStoreOpenApiRepository() {
+        var openApiRepository = new OpenApiRepository();
+        openApiRepository.getOpenApiSpecifications().add(extPetStoreSpecification);
+        return openApiRepository;
+    }
+
+    @Bean(name="ExtPetApi")
```

Comment:
why the specific uppercase name? isn't bean name given my method name anyway?

Reply:
Yes, this is not really required. Removed it.